### PR TITLE
make meshmate-of-the-year-blog responsive

### PR DESCRIPTION
**Pull request by blossom-babs**
[Blog] Fix responsiveness of blog

This PR fixes: #1773 

Issue Description
The blog Meshmate of the Year 2020 is not responsive for mobile view.

Current Behavior
The blog is now responsive across all view sizes

Snippets
blog responsive two
responsive blog one

Notes for Reviewers
Responsiveness can be tested for multiple devices [here](http://www.responsinator.com/)

Signed commits

[* ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @blossom-babs

**Comment by blossom-babs** [comment]
> Thank you, @blossom-babs

Happy to help, Lee

**Comment by l5io** [comment]
🚀 Preview for commit fea8900e5c11f50567500d817c5b748d749d0537 at: https://60788e265dd1677ad9c802da--layer5ng.netlify.app

File: src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.responsive__blog {
```
Comment (Tanuj22) [inline src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css:1]: This file is not being used right?

**Review by Tanuj22 — COMMENTED** [review]
Looking good! Can you also increase the size of the `Meshmate of the year` image on smaller screens? 
Screenshot from 2021-04-16 00-38-26

File: src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css (hunk 1 above)
Comment (blossom-babs) [reply src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css:1]: It is being used in the meshmate-of-the-year mdx file

Comment (Tanuj22) [reply src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css:1]: But I don't see this file being imported anywhere 🤔

Comment (blossom-babs) [reply src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css:1]: Oh, the css file, it's no longer being used, I already put the styling in the general styles folder.

Also, noted the image review

**Comment by l5io** [comment]
🚀 Preview for commit f46eca6417fe5dcf49701a46ad0da61948085cf2 at: https://607c6ede1d8fad2fa25fa9bb--layer5ng.netlify.app

**Review by Tanuj22 — APPROVED** [review]
Thank you @blossom-babs

**Pull request merged**